### ISSUE TITLE
Support for more languages

### ISSUE THREAD

**Issue by tage64** [grantshandy/libretranslate-rs#9]
I see there are much more languages available on libretranslate.com than supported in this crate. Is there any reason for that, or would it be possible to add support for those less common languages as well?

**Comment by grantshandy** [comment]
Yes, it is possible. The reason that there are more languages on libretranslate is that they have updated it since I have edited this repository. I will try to update it tonight so it will include the other languages. I'm also quite a bit more fluent in rust so it should work much better.
